Resources/Prototypes/_Goobstation/Entities/Clothing/Shoes/boots.yml

```
@@ -226,3 +228,14 @@
     - WhitelistChameleon
+
+- type: entity
+  parent: [ClothingShoesMilitaryBase, ClothingShoesBaseWinterBoots, BaseSecurityContraband]
+  id: ClothingShoesWinterCombatBoots
+  name: snowjacks boots
+  description: design to be use in winter on active combat field.
+  components:
+  - type: Sprite
+    sprite: _Goobstation/Clothing/Shoes/Boots/winterbootsjackboot.rsi
+  - type: Clothing
+    sprite: _Goobstation/Clothing/Shoes/Boots/winterbootsjackboot.rsi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)